### PR TITLE
ko-trans: concepts/overview/working-with-objects/names

### LINKED ISSUE #11728: Translate concepts/overview/working-with-objects/names in Korean
**This is a...** 

- [x] Feature Request
- [ ] Bug Report

**Problem:**
No concepts/overview/working-with-objects/names in Korean 

**Proposed Solution:**
Translate concepts/overview/working-with-objects/names in Korean.

**Page to Update:**
https://kubernetes.io/...

/language ko

### PR DESCRIPTION
**Pull request by gochist**
fixes #11728

### CODE REVIEW

File: content/ko/docs/reference/glossary/name.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/names
+short_description: >
+  `/api/v1/pods/some-name`과 같이, 리소스 URL에서 오브젝트를 가리키는 클라이언트 제공 문자열.
+
+aka: 
+tags:
+- fundamental
+---
+ `/api/v1/pods/some-name`과 같이, 리소스 URL에서 오브젝트를 가리키는 클라이언트 제공 문자열
```
Comment (ianychoi) [inline content/ko/docs/reference/glossary/name.md]: Q: Is a period (".") missing?

Comment (gochist) [reply content/ko/docs/reference/glossary/name.md]: @ianychoi Thank you! I addressed your review.

**Pull request merged**